### ISSUE TITLE
Quantization doesn't work?

### ISSUE THREAD

**Issue by praneet195** [yhhhli/BRECQ#31]
Hi,

So I tried running your code on CIFAR-10 with a pre-trained ResNet50 model. I've attached the code below.
My accuracy however does not come nearly as close to the float model which is around 93% but  after quanitzation: I get: 

- Accuracy of the network on the 10000 test images: 10.0 % top5: 52.28 %

Please help me with this. The code is inside the zip file.

[main_cifar.zip](https://github.com/yhhhli/BRECQ/files/8353533/main_cifar.zip)
s

**Comment by mj0219** [comment]
> Hi,
> 
> So I tried running your code on CIFAR-10 with a pre-trained ResNet50 model. I've attached the code below. My accuracy however does not come nearly as close to the float model which is around 93% but after quanitzation: I get:
> 
> * Accuracy of the network on the 10000 test images: 10.0 % top5: 52.28 %
> 
> Please help me with this. The code is inside the zip file.
> 
> [main_cifar.zip](https://github.com/yhhhli/BRECQ/files/8353533/main_cifar.zip) s

I encountered the same issue when using my pretrained model resnet18 on CIFAR-10 for experiments. The accuracy is only about ten percent. Have you resolved this?
